### PR TITLE
Don't overwrite OCAMLPATH in Makefile

### PR DESCRIPTION
**Pull request by pierrebeaucamp**
This is a very minor change in `ulib/Makefile` to append a relative path to the already existing `OCAMLPATH` environment variable instead of overwriting it.

This might be pretty specific to my development environment as I use [Nix](https://nixos.org/) to manage my package dependencies instead of OPAM or similar. Nix works fine except that it utilizes `OCAMLPATH` to set all references to dependencies. For example, my `OCAMLPATH` might look like this:

```
echo $OCAMLPATH
/nix/store/b95kblmp86cpgcan6a42nkp9xh4m56ja-ocaml4.04.2-batteries-2.8.0/lib/ocaml/4.04.2/site-lib/:/nix/store/srvd57r3hb71pg9af2j91jyirbhwcrak-ocaml-findlib-1.7.3/lib/ocaml/4.04.2/site-lib/:/nix/store/vxawlas7bkw490359gp6dkxcn084lzkl-menhir-20170712/lib/ocaml/4.04.2/site-lib/:/nix/store/3r77nh59r4kbv68rd467kicvmaby4m9n-ocamlbuild-0.11.0/lib/ocaml/4.04.2/site-lib/:/nix/store/slb4xhbkbgnjljzlnvr9fj6akzr5k9vs-ocaml4.04.2-ocaml-migrate-parsetree-1.0.7/lib/ocaml/4.04.2/site-lib/:/nix/store/6fb112sjmlpc6kj4rh8ihmkdh7brj1x3-ocaml4.04.2-result-1.2/lib/ocaml/4.04.2/site-lib/:/nix/store/d1msrb46pr2v2pqfg5li04scpkws5ms0-ocaml4.04.2-pprint-20171003/lib/ocaml/4.04.2/site-lib/:/nix/store/y1ilzgklqjlrbwf6lhxffsbvr14vw1ph-ocaml-ppx_deriving-4.2.1/lib/ocaml/4.04.2/site-lib/:/nix/store/dkfih6q714g1k66j2q554n926w7vlcis-ocaml4.04.2-ppx_derivers-1.0/lib/ocaml/4.04.2/site-lib/:/nix/store/qsf554kzdhqm816gwb41xgfdzimbv5x5-ocaml4.04.2-ppx_tools-unstable-20161114/lib/ocaml/4.04.2/site-lib/:/nix/store/0nrlnxrjr63pbay48w8zw4zncmbq8l2r-ocaml4.04.2-ppx_deriving_yojson-3.1/lib/ocaml/4.04.2/site-lib/:/nix/store/gghb0wzmlsk50srsnyfrvfyhlfpmvwg3-ocaml4.04.2-yojson-1.4.0/lib/ocaml/4.04.2/site-lib/:/nix/store/znmj855ccl5mgmvfbh4cx6qrc28qlnd5-cppo-1.5.0/lib/ocaml/4.04.2/site-lib/:/nix/store/vcy4cdvlk3fpkfwpn73nwajjlc5lg8ib-easy-format-1.2.0/lib/ocaml/4.04.2/site-lib/:/nix/store/rsj4f6h8m2wg5rj0xhnx7bcdsxrpn258-ocaml4.04.2-biniou-1.2.0/lib/ocaml/4.04.2/site-lib/:/nix/store/j6x5hp7sq12ialipkyqpn4hawkisaqnk-ocaml4.04.2-process-0.2.1/lib/ocaml/4.04.2/site-lib/:/nix/store/hw6jafm1xil0k5qlmk0h13ymyan6awkk-ocaml4.04.2-stdint-0.3.0/lib/ocaml/4.04.2/site-lib/:/nix/store/w8jy066bavdwcq66wlidcdk7scv75nxy-ocaml4.04.2-ulex-1.2/lib/ocaml/4.04.2/site-lib/:/nix/store/m82nkhlzb1hcn8s23k55s4gjbsjnhxv6-camlp4-4.04+1/lib/ocaml/4.04.2/site-lib/:/nix/store/79jn3jh9m21vck6a32ycnd4r11vmkbch-zarith-1.4.1/lib/ocaml/4.04.2/site-lib/
```

Long story short, when the Makefile sets a custom `OCAMLPATH`, I practically "loose" all my dependencies and I can't build the project anymore. This small patch fixes that problem for any Nix users, or anyone with a custom `OCAMLPATH`.

### CODE REVIEW

**Comment by msprotz** [comment]
Does it work for you if you do

```
OCAMLPATH := ../../../bin/:$(OCAMLPATH)
```

?

**Comment by pierrebeaucamp** [comment]
Yes, thank you that worked as well.
